pkg/helm/helm_helpers.go

```diff
@@ -32,14 +33,21 @@ type k8sObj struct {
 type K8sObjectType string
 
 type Component struct {
-	k8sType K8sObjectType
-	name    string
+	k8sType      K8sObjectType
```

Comment:
I don't see a use case to have this field in the struct, should we remove this?

Reply:
Done